x/programs/cmd/simulator/src/lib.rs

```diff
@@ -199,15 +219,9 @@ pub struct Client {
     path: &'static str,
 }
 
-impl Default for Client {
-    fn default() -> Self {
-        Self::new()
-    }
-}
-
 impl Client {
     #[must_use]
-    pub fn new() -> Self {
+    pub fn new() -> Result<Self, Box<dyn Error>> {
```

Comment:
I don't love this, I would rather just panic

Reply:
Yeah I did some changes and forgot to remove this one, thanks!